### ISSUE TITLE
Support for install specific version

### ISSUE THREAD

**Issue by vchung-nz** [shortdudey123/chef-gluster#95]
Is there any interest in adding an optional attribute to "pin" the client / server to a specific version?  Something like this for client_install.rb
```
include_recipe 'gluster::repository' unless node['gluster']['repo'] == 'private'

# Install the client package
package node['gluster']['client']['package'] do
  version node['gluster']['client']['version']
end
```

**Comment by shortdudey123** [comment]
Right now the version is already pinned to a minor version, but does not go so far as to pin patch versions

You can use `node['gluster']['version']`, but you would also need to modify the repo recipe since it assumes the latest patch version as well as `server_install`

**Comment by vchung-nz** [comment]
Yes, but node['gluster']['version'] is only apply if using public repo.  There is no control within the package resource if using a private repo, hence the suggested edit.
But I am not sure how often people run into that problem :smile:

**Comment by shortdudey123** [comment]
Correct
